source/includes_main/_enrollments.md

```diff
@@ -87,7 +87,7 @@ Attribute | Description
 id | **integer** <br />A unique identifier for the enrollment
 record_id | **integer** <br />The unique identifier of the enrolled record
 record_type | **string** <br />The type of the enrolled record (organization, deal or contact)
-disenrolled_at | **datetime** <br />The date and time when the record has been leaved the workflow in ISO 8601 format with timezone
+disenrolled_at | **datetime** <br />The date and time when the record has been leaved the automation in ISO 8601 format with timezone
```

Comment:
It was there before the change but I'm not sure this is correct English: `… has been leaved the automation …`

Reply:
Nice catch, nothing is correct with this `has been leaved` 🤦‍♂️